src/components/header/Header.jsx

```diff
@@ -2,7 +2,8 @@ import React from "react";
 import "./header.css";
 import { Link } from "react-router-dom";
 import { logo, ProfileIcon, HamburgerIcon } from "../../assets";
-import { useAuth } from "../../context/auth-context";
+import { useAuth } from "../../context";
+import { SearchBar } from "../searchBar/SearchBar";
```

Comment:
Suggestion:- You can use absolute path instead of the relative path as it increases the code readability.

Reply:
Okay, will make the changes in future PRs